pontoon/teams/static/js/info.js

```diff
@@ -0,0 +1,48 @@
+/* global $ */
+$(function() {
+  var container = $('#main .container');
+
+  container.on('click', '.edit-info', function(e) {
+    e.preventDefault();
+    var infoBox = container.find('.info');
+    var content = infoBox.html();
+    var textArea = container.find('.read-write-info textarea').val($.trim(content));
+    container.find('.edit-info').hide();
+    container.find('.read-only-info').hide();
+    container.find('.read-write-info').toggleClass('hidden');
+    textArea.focus();
+  });
+
+  container.on('click', '.cancel', function(e) {
+    e.preventDefault();
+    container.find('.edit-info').show();
+    container.find('.read-only-info').show();
+    container.find('.read-write-info').toggleClass('hidden');
+    return false;
```

Comment:
`return false` within a jQuery event handler is the same as calling both `e.preventDefault` and `e.stopPropagation`. Why do we need to do this when the handler finishes executing its code? Same goes for the `.save` button handler.